server/dive_utils/serializers/viame.py

```diff
@@ -305,10 +295,27 @@ def load_csv_as_tracks_and_attributes(
             create_attributes(metadata_attributes, test_vals, 'track', key, val)
         for (key, val) in attributes.items():
             create_attributes(metadata_attributes, test_vals, 'detection', key, val)
+
+    trackarr = tracks.items()
+
+    if imageMap and len(missingImages) and len(missingImages) != len(trackarr):
```

Comment:
isn't this assuming you are only dealing with detections and not tracks?
add in a test probably for tracks that have all `missingImages`.